packages/jazz-tools/src/tools/coValues/coMap.ts

```diff
@@ -571,6 +571,19 @@ class CoMapJazzApi<M extends CoMap> extends CoValueJazzApi<M> {
     return getCoValueOwner(this.coMap);
   }
 
+  /**
+   * Check if a key is defined in the CoMap.
+   *
+   * This check does not load the referenced value or validate permissions.
+   *
+   * @param key The key to check
+   * @returns True if the key is defined, false otherwise
+   * @category Content
+   */
+  has(key: CoKeys<M>): boolean {
+    return this.raw.get(key) !== undefined;
```

Comment:
If we want to mimic the behavoir of `Map.prototype.has` we should return false for 

```ts
map.$jazz.set(key, undefined);
map.$jazz.has(key)
```

An implementation of this would be:
```suggestion
   const entry = this.raw.getRaw(key);

    if (entry?.change === undefined || entry.change.op === "del") {
      return false;
    }

    return true;
```

Reply:
I guess that's what you meant? Because your proposed implementation returns true in this case